packages/app-desktop/gui/NoteList/NoteList.tsx

```diff
@@ -453,6 +453,14 @@ const NoteListComponent = (props: Props) => {
 		};
 	}, []);
 
+	useEffect(() => {
+		const noteItem = Object.values<any>(itemAnchorRefs_.current)[0]?.current;
+		const actualItemHeight = noteItem?.getHeight() ?? 0;
+		if (actualItemHeight >= 8) { // To avoid generating too many narrow items
+			setItemHeight(actualItemHeight);
+		}
+	});
```

Comment:
Please add a comment to explain why this is needed

Reply:
Done.